src/views/TournamentDetail.vue

```diff
@@ -1,239 +1,349 @@
+      id="infos"
+      :style="{ backgroundImage: `url(${tournament?.logo})` }"
+      class="grid grid-rows-[min-content_1fr] bg-gray-500 bg-cover bg-center bg-blend-multiply"
+      :class="{ hidden: !sections.info[0] }"
+    >
+      <h2 class="font mx-auto my-4 w-3/4 text-center text-3xl font-bold">
+        {{ tournament?.description }}
+      </h2>
 
-            <fa-awesome-icon size="5x" icon="fa-solid fa-circle-plus" />
-            <p class="text-center text-4xl">inscrire son équipe</p>
+      <div class="grid gap-7 md:grid-cols-3">
+        <div>
+          <div class="flex w-full flex-col items-center md:my-24">
+            <h3 class="mb-6 text-4xl">
+              Format
+            </h3>
+            <svg class="w-2/3 fill-white" viewBox="0 0 12 8" xmlns="http://www.w3.org/2000/svg">
+              <desc>
+                Z tetromino with tournament format information
+              </desc>
+              <polygon points="0,0 8,0 8,4 12,4 12,8 4,8 4,4 0,4" fill="#fd5e96"/>
+              <rect fill="#d14d7b" x="0" y="3.5" width="4" height="0.5"/>
+              <rect fill="#d14d7b" x="4" y="7.5" width="8" height="0.5"/>
+              <text v-if="(tournament?.game as Game).players_per_team === 1" font-size="1" text-anchor="middle" x="4" y="2.25">
+                {{ tournament?.maxTeam }} joueurs
+              </text>
+              <text v-else font-size="1" text-anchor="middle" x="4" y="1.25">
+                <tspan>{{ tournament?.maxTeam }} équipes</tspan>
+                <tspan text-anchor="middle" x="4" y="2.75">
+                  de {{ (tournament?.game as Game).players_per_team }} joueurs
+                </tspan>
+              </text>
+              <text font-size="1" text-anchor="middle" x="8" y="5.25">
+                {{ Number(tournament?.player_price_online) }}€ / joueur
+              </text>
+              <text font-size="1" text-anchor="middle" x="8" y="6.75">
+                {{ Number(tournament?.manager_price_online) }}€ / manager
+              </text>
+            </svg>
           </div>
-        </a>
-      -->
+        </div>
+
+        <div>
+          <div class="flex w-full flex-col items-center">
+            <h3 class="mb-6 text-4xl">
+              Cashprize
+            </h3>
+            <svg class="w-2/3 fill-white" viewBox="0 0 12 8" xmlns="http://www.w3.org/2000/svg">
+              <desc>
+                T tetromino with tournament cashprize information
+              </desc>
+              <polygon points="0,8 0,4 4,4 4,0 8,0 8,4 12,4 12,8" fill="#45cae0"/>
+              <rect x="0" y="7.5" width="12" height="0.5" fill="#0aadbf"/>
+              <text font-size="2" text-anchor="middle" x="6" y="2">
+                🥇
+              </text>
+              <text font-size="1" text-anchor="middle" x="6" y="3.25">
+                {{ Number(tournament?.cashprizes[0]) }} €
+              </text>
+              <text font-size="2" text-anchor="middle" x="2" y="6">
+                🥈
```

Comment:
we need to load an emoji font somehow in case the end user doesn't has it

Reply:
For now, we load the OpenMoji font (60dfb99c34a44582c8c623150e9c6b419f29785d) but maybe later we could change for https://icones.js.org/.